bigquery_etl/backfill/utils.py

```diff
@@ -101,6 +101,26 @@ def get_backfill_backup_table_name(qualified_table_name: str, entry_date: date)
     return f"{BACKFILL_DESTINATION_PROJECT}.{BACKFILL_DESTINATION_DATASET}.{cloned_table_id}"
 
 
+def validate_depends_on_past(sql_dir, qualified_table_name) -> bool:
+    """
+    Check if the table depends on past.
+
+    Managed backfills currently do not support tables that depends on past.
+    """
+    project, dataset, table = qualified_table_name_matching(qualified_table_name)
+    dataset_path = Path(sql_dir) / project / dataset
+    table_metadata_path = dataset_path / table / METADATA_FILE
+
+    table_metadata = Metadata.from_file(table_metadata_path)
+
+    depends_on_past = False
+
+    if "depends_on_past" in table_metadata.scheduling:
+        depends_on_past = table_metadata.scheduling["depends_on_past"]
+
+    return not depends_on_past
```

Comment:
The double negative return value is slightly confusing... this is clearer to me, but feel free to ignore.
```suggestion
    if "depends_on_past" in table_metadata.scheduling:
        return not table_metadata.scheduling["depends_on_past"]  # skip if depends_on_past

    return True
```